src/utils/utils.js

```diff
@@ -321,10 +321,31 @@ const mapItemsToKeys = (items, { keyPath, getKeyPath }) =>
     {}
   )
 
+/**
+ * Function to calculate the percentage change between two numbers.
+ * @param {number} originalValue Original number
+ * @param {number} newValue New number
+ *
+ * @example
+ * // Calc percentage change for originalValue: 12 & newValue: 10
+ * calcPercentageChange(12, 10)
+ * //=> -17.67
+ */
+const calcPercentageChange = (originalValue, newValue) => {
+  if (originalValue === 0) {
```

Comment:
What will we do if `originalValue`, which came from a server, is equal `0`? Watch how the app crashes?

Reply:
@DmitriVanGuard 
can I instead return `-newValue` from here? or It will complicate the function's behaviour?

Reply:
@DmitriVanGuard Have added a TODO to fix it later. It's not a blocker at the moment, because the `PercentageChange` component itself also render `NaN` if invalid number is passed as prop

Reply:
@DmitriVanGuard I've added a TODO to fix it later. It's not a blocker ATM, because the `PercentageChange` component itself render `NaN` if invalid number is passed as props